### DISCUSSION TITLE
Should AdobeAcrobatReaderDC support all of the available languages?

### DISCUSSION THREAD

**Discussion by aaronparker** [EUCPilots/evergreen-module#144] [General]
Currently `AdobeAcrobatReaderDC` supports a subset of the full language list offered by Adobe below. Should we add the full list of supported languages. Note that this will increase the time it takes for the query to run.

```
Basque
Chinese (Simplified)
Chinese (Traditional)
Catalan
Croatian
Czech
Danish
Dutch
English
English (UK)
Finnish
French
German
Hungarian
Italian
Japanese
Korean
Norwegian
Polish
Portuguese
Romanian
Russian
Slovakian
Slovenian
Spanish
Swedish
Turkish
Ukrainian
```

**Comment by msfreaks** [comment]
I would say yes.
That being said, I only use the module in unattended automated processes. If that process runs a couple of minutes longer it won't matter for me.

**Comment by aaronparker** [comment]
I tested this out earlier and found the function to go from 17 second to 27 seconds so not such a big difference. 

I did find that the data returned with Portuguese and Chinese languages has an issue, so I'll exclude those languages. I'm starting a known issues page in the documentation.
